pyqtgraph/parametertree/parameterTypes.py

```diff
@@ -231,11 +231,11 @@ def updateDisplayLabel(self, value=None):
             value = self.param.value()
         opts = self.param.opts
         if isinstance(self.widget, QtWidgets.QAbstractSpinBox):
-            text = asUnicode(self.widget.lineEdit().text())
+            text = str(self.widget.lineEdit().text())
```

Comment:
We can be assured this will return a `str` as well: http://doc.qt.io/qt-5/qlineedit.html#text-prop